mesheryctl/internal/cli/root/mesh/mesh.go

```diff
@@ -21,11 +21,15 @@ var MeshCmd = &cobra.Command{
 // Lifecycle management of service meshes
 mesheryctl mesh 
 	`,
-	Args: cobra.MinimumNArgs(1),
+
 	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
+			return errors.New(utils.MeshError(fmt.Sprintln("requires at least 1 arg(s), only received 0 ")))
+		}
```

Comment:
I see that you're trying to implement the docker-compose style of error referencing, which is good. My suggestion is you can add the command in the statement too.

Eg: `Error: mesheryctl mesh requires at least 1 arg(s), only received 0`

The style can be changed a bit... I'll let you know what kind to use after some search